Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -1017,7 +1017,7 @@ routing
             '*' => 'default',
         ];
 
-    You can set a different transport for a specific message, for example:
+    You can set a different transport for a specific message (additional to the above), for example:
 
     ..  code-block:: php
 
```

Comment:
$GLOBALS['TYPO3_CONF_VARS']['SYS']['messenger']['routing'][\MyVendor\MyExtension\Queue\Message\DemoMessage::class] = 'doctrine';
        
Sorry, couldn't write it better, it didn't let me

Reply:
Aah, you are right, missed it, will adjust it.

Reply:
Done